Jenkinsfile

```diff
@@ -69,11 +78,14 @@ onlyOnMaster {
     milestone()
     stage("prod Deploy") {
         parallel(
-            us: {
+            us_h: {
                 deployApp(image: image, app: "h", env: "prod", region: "us-west-1")
             },
-            ca: {
-		// Workaround to ensure all parallel builds happen. See https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400
+            us_ws: {
+                sleep 2
+                deployApp(image: image, app: "h-websocket", env: "prod", region: "us-west-1")
+            },
+            ca_h: {
                 sleep 2
                 deployApp(image: image, app: "h-ca", env: "prod", region: "ca-central-1")
             }
```

Comment:
The parallel block used in the `prod` stage has been expanded.

3x `deployApp` calls are being made. The first deploys `h` in `us-west-1` as we always have, and the second takes care of `h-websocket` in `us-west-1`. The final call deploys the `h` service running in Canada.

Comment removed, and added to the `qa` stage. See previous remarks for more information.